xdrip-get-entries.sh

```diff
@@ -133,7 +133,7 @@ main()
 
   fake_meter
 
-  if [ "$mode" != "Stopped" ]; then
+  if [ "$state" != "Stopped" ]; then
```

Comment:
maybe just add && [ "$mode" != "expired" ]  ?

Reply:
I tweaked this to:   || [ "$mode" != "not-expired" ] so that it will proceed if it's either not stopped or if it's not not-expired (thinking ahead if we add an auto-expire mode)